packages/web/src/components/ui/ChipGroup/ChipGroup.tsx

```diff
@@ -16,15 +19,23 @@ export default function ChipGroup({
   size = 'medium',
   font = 'monospace',
   fontSize = 'regular',
+  loading,
 }: Props) {
   return (
     <div className={styles.chipsWrapper}>
       <div className={styles.chips}>
-        {chips.map((chip) => (
-          <Chip key={chip} className={styles.chip} size={size} font={font} fontSize={fontSize}>
-            {chip}
-          </Chip>
-        ))}
+        {loading
+          ? repeat(
+              4,
+              <Skeleton variant='rounded' className={styles.chip}>
+                <Chip>Lorem ipsum</Chip>
```

Comment:
Are we gonna render skeleton elements in SSR? If so, I'd like to not have any dummy text inside skeletons. If not, we probably must ensure that the dummy text will not accidentally pass into generated html.